controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/responses/deployment-overview.json

```diff
@@ -6,19 +6,47 @@
       "type": "instance",
       "dependencies": [],
       "declared": true,
-      "instance": "instance1"
+      "instance": "instance1",
+      "readyAt": 0,
+      "deploying": {
+        "application": {
+          "build": 4,
+          "compileVersion": "6.1.0",
+          "sourceUrl": "repository1/tree/commit1",
+          "commit": "commit1"
+        }
+      },
+      "latestVersions": {
+        "platform": {
+          "platform": "7.164.0",
+          "at": "(ignore)",
+          "upgrade": false,
+          "blockers": []
+        },
+        "application": {
+          "application": {
+            "build": 4,
+            "compileVersion": "6.1.0",
+            "sourceUrl": "repository1/tree/commit1",
+            "commit": "commit1"
+          },
+          "at": 1000,
```

Comment:
What is this?

Reply:
millis at which the version was built (application) or of last commit (Vespa).